nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -120,12 +123,30 @@ public ValidationResult validate(final String subject, final String value, final
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
         .build();
 
+    static final String JSON_TYPE_EXTENDED = "Extended";
+    static final String JSON_TYPE_NORMAL   = "Normal";
+    static final AllowableValue JSON_EXTENDED = new AllowableValue(JSON_TYPE_EXTENDED, "Extended JSON",
+            "Use MongoDB's \"extended JSON\". This is the JSON generated with toJson() on a MongoDB Document from the Java driver");
+    static final AllowableValue JSON_NORMAL = new AllowableValue(JSON_TYPE_NORMAL, "Normal JSON",
```

Comment:
Should this be "Standard JSON" rather than "Normal JSON"? Both terms are overloaded, but the description refers to it as "standard JSON". Also mention of Jackson is an implementations-specific detail and not necessary for the user documentation (plus if the underlying impl changes we'd have to remember to change this too).